<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
      <title>Kim Rutherford - Software Development and Bioinformatics</title>
  <style type="text/css">code{white-space: pre;}</style>
  <!--[if lt IE 9]>
    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->
        <link rel="stylesheet" href="stylesheets/style.css">
    </head>
<body>
  <header itemscope itemtype="http://http://schema.org/Person" class="">
    <div id="title" class="">
            <h1 class="fullname">
        <span itemprop="givenName">Kim</span>
        <span itemprop="familyName">Rutherford</span>
      </h1>
      <h2 class="title">Software Development and Bioinformatics</h2>
    </div>
        <ul class="details">
      <!-- phone -->
            <!-- mobile -->
                        <li><a class='mobile' href="phoneto:+64 212508293">+64 212508293</a></li>
                        <!-- fax -->
            <!-- email -->
                  <li><a href="mailto:kmr@bioinformatics.nz">kmr@bioinformatics.nz</a></li>
                  <!-- homepage -->
                  <li><a href="https://kmr.bioinformatics.nz" itemprop="url" title="homepage">https://kmr.bioinformatics.nz</a></li>
                  <!--if(address)-->
                  <li itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
                <span itemprop="addressLocality">Dunedin, </span>
                        <span itemprop="addressCountry">New Zealand</span>
              </li>
                  <!--endif-->
    </ul>
  </header>

  
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-131009466-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    gtag('config', 'UA-131009466-1');
  </script>
  <section id="profile" class="level2">
  <h2>Profile</h2>
  <p>I am a software developer and bioinformatician. For the last 20 years I have been helping biologists to capture, analyse and interpret genomic data.</p>
  </section>
  <section id="skills-and-experience" class="level2">
  <h2>Skills and Experience</h2>
  <ul>
  <li>Software development: web, desktop and server</li>
  <li>Database administration and maintenance</li>
  <li>Genome assembly, annotation and analysis</li>
  <li>Processing, management and analysis of next generation sequencing data</li>
  <li>Systems administration including software and database installation and configuration</li>
  </ul>
  </section>
  <section id="work-experience" class="level2">
  <h2>Work experience</h2>
  <dl>
  <dt>June 2010 – Present</dt>
  <dd><p><strong>Programmer</strong> <em>S. pombe genome database (PomBase)</em><br />
  Department of Biochemistry, University of Cambridge</p>
  <dl>
  <dt>Tasks</dt>
  <dd>web and database programming
  </dd>
  <dd>system administration
  </dd>
  <dt>Major projects</dt>
  <dd><a href="https://curation.pombase.org">Canto</a> - a community curation tool
  </dd>
  <dd><a href="https://www.pombase.org">PomBase v2</a>
  </dd>
  </dl>
  </dd>
  <dt>Sept 2012 – Dec 2016</dt>
  <dd><p><strong>Scientific Officer</strong> (part time) <em>Gemmell lab</em><br />
  Department of Anatomy, University of Otago</p>
  <dl>
  <dt>Tasks</dt>
  <dd>bioinformatics support for the group
  </dd>
  <dd>genome and transcriptome assembly and analysis
  </dd>
  <dd>mentoring / training of group members in bioinformatics
  </dd>
  <dd><a href="https://en.wikipedia.org/wiki/Tuatara">tuatara</a> <a href="http://gemmell-lab.otago.ac.nz/research-projects/current-research-projects/1-sequencing-a-living-fossil-the-tuatara-genome-project">genome project</a>
  </dd>
  </dl>
  </dd>
  <dt>Feb 2009 – June 2010</dt>
  <dd><p><strong>Computer Associate</strong> <em>Baulcombe Group</em><br />
  Department of Plant Sciences, University of Cambridge</p>
  <dl>
  <dt>Tasks</dt>
  <dd>bioinformatics support
  </dd>
  <dd>creation of a short read analysis pipeline
  </dd>
  </dl>
  </dd>
  <dt>Sept 2007 – Feb 2009</dt>
  <dd><p><strong>Programmer</strong> <em>modENCODE Data Coordination Center</em><br />
  Cambridge Systems Biology Centre</p>
  <dl>
  <dt>Tasks</dt>
  <dd>helped create the data warehouse for the modENCODE project:<br />
  <a href="http://intermine.modencode.org/">modMine</a>
  </dd>
  </dl>
  </dd>
  <dt>Feb 2004 – Sept 2007</dt>
  <dd><p><strong>Programmer</strong> <em>FlyMine/InterMine group</em><br />
  Department of Genetics, University of Cambridge</p>
  <dl>
  <dt>Tasks</dt>
  <dd>Java programming for the <a href="http://intermine.org">InterMine</a> project
  </dd>
  </dl>
  </dd>
  <dt>Oct 2001 – Feb 2004</dt>
  <dd><p><strong>Principal Computer Programmer</strong> <em>Pathogen Sequencing Unit</em><br />
  The Sanger Institute</p>
  <dl>
  <dt>Tasks</dt>
  <dd>programming and bioinformatics support
  </dd>
  <dt>Major projects</dt>
  <dd><a href="http://www.sanger.ac.uk/science/tools/artemis">Artemis</a> - a genome analysis and annotation tool
  </dd>
  <dd><a href="http://www.sanger.ac.uk/science/tools/artemis-comparison-tool-act">ACT</a> - a genome comparison viewer
  </dd>
  </dl>
  </dd>
  <dt>Apr 1999 – Oct 2001</dt>
  <dd><strong>Senior Computer Programmer</strong> <em>Pathogen Sequencing Unit</em><br />
  The Sanger Institute
  </dd>
  <dt>Sept 1998 – Apr 1999</dt>
  <dd><strong>Computer Programmer</strong> <em>Pathogen Sequencing Unit</em><br />
  The Sanger Institute
  </dd>
  </dl>
  </section>
  <section id="publications" class="level2">
  <h2>Publications</h2>
  <div style="float: right; font-size: 90%;">
  <span style="color: grey">●</span> <a href="http://www.scopus.com/authid/detail.uri?authorId=7007171631">Scopus profile</a> <span style="color: grey">●</span> <a href="https://scholar.google.co.nz/citations?hl=en&amp;user=63wyizYAAAAJ&amp;view_op=list_works&amp;sortby=pubdate">Google Scholar</a> <span style="color: grey">●</span> <a href="https://www.researcherid.com/rid/Y-7995-2018">ResearcherID</a> <span style="color: grey">●</span> <a href="http://orcid.org/0000-0001-6277-726X">ORCID</a> <span style="color: grey">●</span> <a href="http://europepmc.org/authors/0000-0001-6277-726X">Europe PMC</a>
  </div>
  <dl>
  <dt>Dec 2018</dt>
  <dd><strong>Stress, novel sex genes and epigenetic reprogramming orchestrate socially-controlled sex change</strong><br />
  E.V. Todd, O. Ortega-Recalde, H. Liu, M.S. Lamm, K.M. Rutherford, H. Cross, M.A. Black, O. Kardailsky, J.A. Graves, T.A. Hore, J.R. Godwin, N.J. Gemmell<br />
  <em>bioRxiv pre-print: <a href="https://doi.org/10.1101/481143">10.1101/481143</a></em>
  </dd>
  <dt>Nov 2018</dt>
  <dd><strong>Hidden in plain sight: What remains to be discovered in the eukaryotic proteome?</strong><br />
  V. Wood, A. Lock, M. Harris, K. Rutherford, J. Bahler, S Oliver<br />
  <em>Open Biology. <a href="https://royalsocietypublishing.org/doi/full/10.1098/rsob.180241">10.1098/rsob.180241</a></em>
  </dd>
  <dt>Oct 2018</dt>
  <dd><strong>RNAcentral: a hub of information for non-coding RNA sequences</strong><br />
  The RNAcentral Constortium<br />
  <em>Nucleic Acids Research. <a href="https://doi.org/10.1093/nar/gky1034">10.1093/nar/gky1034</a></em>
  </dd>
  <dt>Oct 2018</dt>
  <dd><strong>The Gene Ontology Resource: 20 years and still GOing strong</strong><br />
  The Gene Ontology Consortium<br />
  <em>Nucleic Acids Research. <a href="https://doi.org/10.1093/nar/gky1055">10.1093/nar/gky1055</a></em>
  </dd>
  <dt>Oct 2018</dt>
  <dd><strong>PomBase 2018: user-driven reimplementation of the fission yeast database provides rapid and intuitive access to diverse, interconnected information</strong><br />
  A. Lock, K. Rutherford, M.A. Harris, J Hayles, S.G. Oliver, J. Bähler, V. Wood<br />
  <em>Nucleic Acids Research. <a href="https://doi.org/10.1093/nar/gky961">10.1093/nar/gky961</a></em>
  </dd>
  <dt>Aug 2018</dt>
  <dd><strong>Evolutionary history of the podoplanin gene</strong><br />
  J. Renart, D. San Mauro, A. Agorreta, K. Rutherford, N. J. Gemmell, M Quintanill<br />
  <em>Gene Reports. <a href="https://doi.org/10.1016/j.genrep.2018.08.005">10.1016/j.genrep.2018.08.005</a></em>
  </dd>
  <dt>Aug 2018</dt>
  <dd><strong>Reduced representation sequencing detects only subtle regional structure in a heavily exploited and rapidly recolonizing marine mammal species</strong><br />
  N. Dussex, H.R. Taylor, W.R. Stovall, K. Rutherford, K.G. Dodds, S.M. Clarke, N.J. Gemmell<br />
  <em>Ecology and Evolution. <a href="https://doi.org/10.1002/ece3.4411">10.1002/ece3.4411</a></em>
  </dd>
  <dt>Apr 2018</dt>
  <dd><strong>Adipose Transcriptome Analysis Provides Novel Insights into Molecular Regulation of Prolonged Fasting in Northern Elephant Seal Pups</strong><br />
  B. Martinez, J. Khudyakov, K. Rutherford, D. Crocker, N. Gemmell, R. Ortiz<br />
  <em>Physiological Genomics. <a href="https://doi.org/10.1152/physiolgenomics.00002.2018">10.1152/physiolgenomics.00002.2018</a></em>
  </dd>
  <dt>Apr 2018</dt>
  <dd><strong><em>De novo</em> draft assembly of the <em>Botrylloides leachii</em> genome provides further insight into tunicate evolution</strong><br />
  S. Blanchoud, K. Rutherford, L. Zondag, N. Gemmell, M Wilson<br />
  <em>Sci Rep. 2018 Apr 3;8(1):5518. <a href="https://doi.org/10.1038/s41598-018-23749-w">10.1038/s41598-018-23749-w.</a></em>
  </dd>
  <dt>Feb 2018</dt>
  <dd><strong>Genetic sex assignment in wild populations using GBS data: a statistical threshold approach</strong><br />
  W. Stovall, H.R. Taylor, M. Black, S. Grosser, K. Rutherford, N.J. Gemmell<br />
  <em>Molecular Ecology Resources. <a href="https://doi.org/10.1111/1755-0998.12767">10.1111/1755-0998.12767</a></em>
  </dd>
  <dt>Dec 2017</dt>
  <dd><strong>Analysis of the genome of the New Zealand giant collembolan (<em>Holacanthella duospinosa</em>) sheds light on hexapod evolution</strong><br />
  C. Wu, M.D. Jordan, R.D. Newcomb, N.J. Gemmell, S. Bank, K. Meusemann, P.K. Dearden, E.J. Duncan, S. Grosser, K. Rutherford, P.P. Gardner, R.N. Crowhurst, B. Steinwender, L.K. Tooman, M.I. Stevens, T.R. Buckley<br />
  <em>BMC Genomics. 2017 Oct 17;18(1):795. <a href="https://doi.org/10.1186/s12864-017-4197-1">10.1186/s12864-017-4197-1.</a></em>
  </dd>
  <dt>Dec 2017</dt>
  <dd><strong>Identification of sex differences in zebrafish (<em>Danio rerio</em>) brains during early sexual differentiation and masculinisation using 17α-methyltestoterone</strong><br />
  S.L.J. Lee, J.A. Horsfield, M.A. Black, K.M. Rutherford, N.J. Gemmell<br />
  <em>Biology of Reproduction, <a href="https://doi.org/10.1093/biolre/iox175">10.1093/biolre/iox175</a></em>
  </dd>
  <dt>Nov 2017</dt>
  <dd><strong>Female mimicry by sneaker males has a transcriptomic signature in both the brain and gonad in a sex changing fish</strong><br />
  E.V. Todd, H. Liu, M.S. Lamm, J.T. Thomas, K. Rutherford, K.C. Thompson, J.R. Godwin, N.J. Gemmell<br />
  <em>Molecular Biology and Evolution, <a href="https://doi.org/10.1093/molbev/msx293">10.1093/molbev/msx293</a></em>
  </dd>
  <dt>July 2017</dt>
  <dd><strong>Male-female relatedness at specific SNP-linkage groups influences cryptic female choice in Chinook salmon (<em>Oncorhynchus tshawytscha</em>)</strong><br />
  C. Gessner, S.L. Johnson, P. Fisher, S. Clarke, K. Rutherford, J. Symonds, N.J. Gemmell<br />
  <em>Proc. R. Soc. B 2017 284 20170853; <a href="https://doi.org/10.1098/rspb.2017.0853">10.1098/rspb.2017.0853.</a></em>
  </dd>
  <dt>July 2017</dt>
  <dd><strong>Histological and transcriptomic effects of 17α-methyltestosterone on zebrafish gonad development</strong><br />
  S.L.J. Lee, J.A. Horsfield, M.A. Black, K. Rutherford, A. Fisher, N.J. Gemmell<br />
  <em>BMC Genomics (2017) 18:557 <a href="https://doi.org/10.1186/s12864-017-3915-z">10.1186/s12864-017-3915-z</a></em>
  </dd>
  <dt>June 2017</dt>
  <dd><strong>PomBase – the scientific resource for fission yeast</strong><br />
  V. Wood, A. Lock, K. Rutherford, M.A. Harris<br />
  <em>Methods in Molecular Biology <a href="https://doi.org/10.17863/CAM.12124">10.17863/CAM.12124</a></em>
  </dd>
  <dt>Jan 2017</dt>
  <dd><strong>PHI-base: A new interface and further additions for the multi-species pathogen-host interactions database.</strong><br />
  Urban, M., Alayne Cuzick, A., Rutherford, K., Irvine, A., Pedro, H., Pant, R., Sadanadan, V., Khamari, L., Billal, S., Mohanty S., and. Hammond-Kosack, K.E.<br />
  <em>Nucleic Acids Research (2017) <a href="https://doi.org/10.1093/nar/gkw1089">10.1093/nar/gkw1089</a></em>
  </dd>
  <dt>Nov 2016</dt>
  <dd><strong>The Gene Ontology Consortium; Expansion of the Gene Ontology knowledgebase and resources.</strong><br />
  The Gene Ontology Consortium<br />
  <em>Nucleic Acids Res 2017; 45 (D1): D331-D338. <a href="https://doi.org/10.1093/nar/gkw1108">10.1093/nar/gkw1108</a></em>
  </dd>
  <dt>Feb 2016</dt>
  <dd><strong>Uncovering the pathways underlying whole body regeneration in a chordate model, <em>Botrylloides leachi</em> using de novo transcriptome analysis</strong><br />
  L. Zondag, K. Rutherford, N. Gemmell and M. Wilson<br />
  <em>BMC Genomics (2016) 17:114 <a href="https://doi.org/10.1186/s12864-016-2435-6">10.1186/s12864-016-2435-6</a></em>
  </dd>
  <dt>Nov 2015</dt>
  <dd><strong>Large-scale transcriptome sequencing reveals novel expression patterns for key sex-related genes in a sex-changing fish</strong><br />
  H. Liu, M. Lamm, K. Rutherford, M. Black, J. Godwin and N. Gemmell<br />
  <em>Biology of Sex Differences (2015) 6:26 <a href="https://dx.doi.org/10.1186%2Fs13293-015-0044-8">10.1186/s13293-015-0044-8</a></em>
  </dd>
  <dt>Jan 2015</dt>
  <dd><strong>Gene Ontology Consortium: going forward</strong><br />
  The Gene Ontology Consortium<br />
  <em>Nucl. Acids Res. (28 January 2015) 43 (D1): D1049-D1056. <a href="https://doi.org/10.1093/nar/gku1179">10.1093/nar/gku1179</a></em>
  </dd>
  <dt>Dec 2014</dt>
  <dd><strong>Molecular evolution of Dmrt1 accompanies change of sex-determining mechanisms in reptilia</strong><br />
  D. E. Janes <em>et al.</em><br />
  <em>Biol Lett. 2014 Dec; 10(12): 20140809. <a href="https://doi.org/10.1098/rsbl.2014.0809">10.1098/rsbl.2014.0809</a></em>
  </dd>
  <dt>Oct 2014</dt>
  <dd><strong>PomBase 2015: updates to the fission yeast database</strong><br />
  M. McDowall, M. Harris, A. Lock, K. Rutherford, D. Staines, J. Bähler, P. Kersey, S. Oliver and V. Wood<br />
  <em>Nucl. Acids Res. (2014) <a href="https://doi.org/10.1093/nar/gku1040">10.1093/nar/gku1040</a></em>
  </dd>
  <dt>June 2014</dt>
  <dd><strong>Improving functional annotation for industrial microbes: A case study with <em>Pichia pastoris</em></strong><br />
  D. Dikicioglu, V. Wood, K. Rutherford, M. McDowall, S. Oliver<br />
  <em>Trends in Biotechnology (2014) <a href="https://doi.org/10.1016/j.tibtech.2014.05.003">10.1016/j.tibtech.2014.05.003</a></em>
  </dd>
  <dt>Feb 2014</dt>
  <dd><strong>Canto: An online tool for community literature curation</strong><br />
  K. Rutherford, M. A. Harris, A. Lock, S. G. Oliver and V. Wood<br />
  <em>Bioinformatics (2014) <a href="https://doi.org/10.1093/bioinformatics/btu103">10.1093/bioinformatics/btu103</a></em>
  </dd>
  <dt>Jan 2013</dt>
  <dd><strong>Gene ontology annotations and resources.</strong><br />
  Gene Ontology Consortium<br />
  <em>Nucleic Acids Res. 2013 Jan;41(Database issue):D530-5. <a href="https://doi.org/10.1093/nar/gks1050">10.1093/nar/gks1050</a></em>
  </dd>
  <dt>Sep 2012</dt>
  <dd><strong>InterMine: a flexible data warehouse system for the integration and analysis of heterogeneous biological data</strong><br />
  R.N. Smith, J. Aleksic, D. Butano, A. Carr, S. Contrino, F. Hu, M. Lyne, R. Lyne, A. Kalderimis, K. Rutherford, R. Stepan, J. Sullivan, M. Wakeling, X. Watkins, G. Micklem<br />
  <em>Bioinformatics. (2012) 28 (23):3163-3165. <a href="https://doi.org/10.1093/bioinformatics/bts577">10.1093/bioinformatics/bts577</a></em>
  </dd>
  <dt>Jan 2012</dt>
  <dd><strong>modMine: flexible access to modENCODE data.</strong><br />
  S. Contrino, R.N. Smith, D. Butano, A. Carr, F. Hu, R. Lyne, K. Rutherford, A. Kalderimis, J. Sullivan, S. Carbon, E.T. Kephart, P. Lloyd, E.O. Stinson, N.L. Washington, M.D. Perry, P. Ruzanov, Z. Zha, S.E. Lewis, L.D. Stein, G. Micklem<br />
  <em>Nucleic Acids Res. 2012 Jan;40(Database issue):D1082-8. <a href="https://doi.org/10.1093/nar/gkr921">10.1093/nar/gkr921</a></em>
  </dd>
  <dt>Jan 2012</dt>
  <dd><strong>PomBase: a comprehensive online resource for fission yeast.</strong><br />
  V. Wood, M.A. Harris, M.D. McDowall, K. Rutherford, B.W. Vaughan, D.M. Staines, M. Aslett, A. Lock, J. Bähler, P.J. Kersey, S.G. Oliver<br />
  <em>Nucleic Acids Res. 2012 Jan;40(Database issue):D695-9. <a href="https://doi.org/10.1093/nar/gkr853">10.1093/nar/gkr853</a></em>
  </dd>
  <dt>Nov 2011</dt>
  <dd><strong>The Gene Ontology: enhancements for 2011.</strong><br />
  Gene Ontology Consortium.<br />
  <em>Nucleic Acids Res. 2012 Jan;40(Database issue):D559-64. <a href="https://doi.org/10.1093/nar/gkr1028">10.1093/nar/gkr1028</a></em>
  </dd>
  <dt>Dec 2010</dt>
  <dd><strong>Identification of Functional Elements and Regulatory Circuits by <em>Drosophila</em> modENCODE.</strong><br />
  The modENCODE Consortium<br />
  <em>Science. 2010 Dec 24;330(6012):1787-97. <a href="https://doi.org/10.1126/science.1198374">10.1126/science.1198374</a></em>
  </dd>
  <dt>Dec 2010</dt>
  <dd><strong>Integrative Analysis of the <em>C. elegans</em> Genome by the modENCODE Project.</strong><br />
  The modENCODE Consortium<br />
  <em>Science. 2010 Dec 24;330(6012):1775-87. <a href="https://doi.org/10.1126/science.1196914">10.1126/science.1196914</a></em>
  </dd>
  <dt>July 2007</dt>
  <dd><strong>FlyMine: an integrated database for <em>Drosophila</em> and <em>Anopheles</em> genomics.</strong><br />
  R. Lyne <em>et al.</em><br />
  <em>Genome Biol. 2007;8(7):R129. <a href="https://doi.org/10.1186/gb-2007-8-7-r129">10.1186/gb-2007-8-7-r129</a></em>
  </dd>
  <dt>Sep 2005</dt>
  <dd><strong>WebACT - An online companion for the Artemis Comparison Tool</strong><br />
  J. Abbott, D. Aanensen, K. Rutherford, S. Butcher and B. Spratt<br />
  <em>Bioinformatics. 2005 Sep 15;21(18):3665-6. <a href="https://doi.org/10.1093/bioinformatics/bti601">10.1093/bioinformatics/bti601</a></em>
  </dd>
  <dt>Aug 2005</dt>
  <dd><strong>ACT: the Artemis comparison tool.</strong><br />
  T. Carver, K. Rutherford, M. Berriman, M-A. Rajandream, B.. Barrell and J. Parkhill<br />
  <em>Bioinformatics. 2005 Aug 15;21(16):3422-3 <a href="https://doi.org/10.1093/bioinformatics/bti553">10.1093/bioinformatics/bti553</a></em>
  </dd>
  <dt>June 2005</dt>
  <dd><strong>A Human-Curated Annotation of the <em>Candida albicans</em> Genome</strong><br />
  Braun <em>et al.</em><br />
  <em>PLoS Genet. 2005 Jul;1(1):36-57. <a href="https://doi.org/10.1371/journal.pgen.0010001">10.1371/journal.pgen.0010001</a></em>
  </dd>
  <dt>Jan 2005</dt>
  <dd><strong>A comprehensive survey of the <em>Plasmodium</em> life cycle by genomic, transcriptomic, and proteomic analyses</strong><br />
  N. Hall <em>et al.</em><br />
  <em>Science 07 Jan 2005: Vol. 307, Issue 5706, pp. 82-86 <a href="https://doi.org/10.1126/science.1103717">10.1126/science.1103717</a></em>
  </dd>
  <dt>Sep 2004</dt>
  <dd><strong>Genomic plasticity of the causative agent of melioidosis, <em>Burkholderia pseudomallei</em>.</strong><br />
  M. Holden <em>et al.</em><br />
  <em>Proc Natl Acad Sci U S A. 2004 Sep 28;101(39):14240-5. <a href="https://doi.org/10.1073/pnas.0403302101">10.1073/pnas.0403302101</a></em>
  </dd>
  <dt>June 2004</dt>
  <dd><strong>Complete genomes of two clinical <em>Staphylococcus aureus</em> strains: Evidence for the rapid evolution of virulence and drug resistance.</strong><br />
  M. Holden <em>et al.</em><br />
  <em>Proc Natl Acad Sci U S A. 2004 Jun 29;101(26):9786-91 <a href="https://doi.org/10.1073/pnas.0402521101">10.1073/pnas.0402521101</a></em>
  </dd>
  <dt>Jan 2004</dt>
  <dd><strong>GeneDB: a resource for prokaryotic and eukaryotic organisms.</strong><br />
  Hertz-Fowler C <em>et al.</em><br />
  <em>Nucleic Acids Res. 2004 Jan 1;32(Database issue):D339-43. <a href="https://doi.org/10.1093/nar/gkh007">10.1093/nar/gkh007</a></em>
  </dd>
  <dt>Nov 2003</dt>
  <dd><strong>The complete genome sequence and analysis of <em>Corynebacterium diphtheriae</em> NCTC13129. </strong><br />
  A. Cerdeño‐Tárraga <em>et al.</em><br />
  <em>Nucleic Acids Res. 2003 Nov 15;31(22):6516-23. <a href="https://doi.org/10.1093/nar/gkg874">10.1093/nar/gkg874</a></em>
  </dd>
  <dt>Aug 2003</dt>
  <dd><strong>The DNA sequence of chromosome I of an African trypanosome: Gene content, chromosome organisation, recombination and polymorphism</strong><br />
  N. Hall <em>et al.</em><br />
  <em>Nucl. Acids Res. (2003) 31 (16): 4864-4873. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC169939/">PMC169939</a></em>
  </dd>
  <dt>June 2003</dt>
  <dd><strong>Viewing and annotating sequence data with Artemis.</strong><br />
  M. Berriman and K. Rutherford<br />
  <em>Brief Bioinform. 2003 Jun;4(2):124-32. <a href="http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.99.6409&amp;rep=rep1&amp;type=pdf">PMID:12846394</a></em>
  </dd>
  <dt>Oct 2002</dt>
  <dd><strong>Genome sequence of the human malaria parasite <em>Plasmodium falciparum</em>.</strong><br />
  J. Malcolm <em>et al.</em><br />
  <em>Nature. 2002 Oct 3;419(6906):498-511. <a href="https://doi.org/10.1038/nature01097">10.1038/nature01097</a></em>
  </dd>
  <dt>Oct 2002</dt>
  <dd><strong>Sequence of <em>Plasmodium falciparum</em> chromosomes 1, 3–9 and 13.</strong><br />
  N. Hall <em>et al.</em><br />
  <em>Nature. 2002 Oct 3;419(6906):527-31. <a href="https://doi.org/10.1038/nature01095">10.1038/nature01095</a></em>
  </dd>
  <dt>May 2002</dt>
  <dd><strong>Complete genome sequence of the model actinomycete <em>Streptomyces coelicolor</em> A3(2).</strong><br />
  S. Bentley <em>et al.</em><br />
  <em>Nature. 2002 May 9;417(6885):141-7. <a href="https://doi.org/10.1038/417141a">10.1038/417141a</a></em>
  </dd>
  <dt>Feb 2002</dt>
  <dd><strong>The genome sequence of <em>Schizosaccharomyces pombe</em>.</strong><br />
  V. Wood <em>et al.</em><br />
  <em>Nature. 2002 Feb 21;415(6874):871-80. <a href="https://doi.org/10.1038/nature724">10.1038/nature724</a></em>
  </dd>
  <dt>Oct 2001</dt>
  <dd><strong>Complete genome sequence of a multiple drug resistant <em>Salmonella enterica</em> serovar Typhi CT18.</strong><br />
  J. Parkhill <em>et al.</em><br />
  <em>Nature. 2001 Oct 25;413(6858):848-52. <a href="https://doi.org/10.1038/35101607">10.1038/35101607</a></em>
  </dd>
  <dt>Oct 2001</dt>
  <dd><strong>Genome sequence of <em>Yersinia pestis</em>, the causative agent of plague.</strong><br />
  J. Parkhill <em>et al.</em><br />
  <em>Nature. 2001 Oct 4;413(6855):523-7. <a href="https://doi.org/10.1038/35097083">10.1038/35097083</a></em>
  </dd>
  <dt>June 2001</dt>
  <dd><strong>A Re-annotation of the <em>Saccharomyces cerevisiae</em> Genome.</strong><br />
  V. Wood, K. M. Rutherford, A. Ivens, M-A Rajandream and B. Barrell<br />
  <em>Comp Funct Genomics. 2001 June; 2(3): 143–154. <a href="https://doi.org/10.1002/cfg.86">10.1002/cfg.86</a></em>
  </dd>
  <dt>Feb 2001</dt>
  <dd><strong>Massive gene decay in the leprosy bacillus.</strong><br />
  S. Cole <em>et al.</em><br />
  <em>Nature. 2001 Feb 22;409(6823):1007-11. <a href="https://doi.org/10.1038/35059006">10.1038/35059006</a></em>
  </dd>
  <dt>Oct 2000</dt>
  <dd><strong>Artemis: sequence visualisation and annotation.</strong><br />
  K. Rutherford, J. Parkhill, J. Crook, T. Horsnell, P. Rice, M-A. Rajandream and B. Barrell<br />
  <em>Bioinformatics. 2000 Oct;16(10):944-5. <a href="https://doi.org/10.1093/bioinformatics/16.10.944">10.1093/bioinformatics/16.10.944</a></em>
  </dd>
  <dt>Sep 2000</dt>
  <dd><strong>Analysis of 114 kb of DNA sequence from fission yeast chromosome 2 immediately centromere‐distal to <em>his5</em>.</strong><br />
  Z. Xiang <em>et al.</em><br />
  <em>Yeast 16: 1405–1411. <a href="https://doi.org/10.1002/1097-0061(200011)16:15%3C1405::AID-YEA625%3E3.0.CO;2-H">full text</a></em>
  </dd>
  <dt>Mar 2000</dt>
  <dd><strong>Complete DNA sequence of a serogroup A strain of <em>Neisseria meningitidis</em> Z2491.</strong><br />
  J. Parkhill <em>et al.</em><br />
  <em>Nature. 2000 Mar 30;404(6777):502-6. <a href="https://doi.org/10.1038/35006655">10.1038/35006655</a></em>
  </dd>
  <dt>Feb 2000</dt>
  <dd><strong>The genome sequence of the food-borne pathogen <em>Campylobacter jejuni</em> reveals hypervariable sequences.</strong><br />
  J. Parkhill <em>et al.</em><br />
  <em>Nature. 2000 Feb 10;403(6770):665-8. <a href="https://doi.org/10.1038/35001088">10.1038/35001088</a></em>
  </dd>
  </dl>
  </section>
  <section id="book-chapters" class="level2">
  <h2>Book Chapters</h2>
  <dl>
  <dt>Apr 2018</dt>
  <dd><strong>PomBase: The Scientific Resource for Fission Yeast</strong><br />
  “Eukaryotic Genomic Databases : Methods and Protocols”<br />
  A Lock, K Rutherford, M.A. Harris, V. Wood<br />
  <em>Methods in Molecular Biology, vol 1757. <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7737-6_4">10.1007/978-1-4939-7737-6_4</a></em>
  </dd>
  </dl>
  </section>

  
  <footer>
    <ul>
    <li><a href="http://twitter.com/kim_rutherford">kim_rutherford</a></li>
    <li><a href="https://github.com/kimrutherford">kimrutherford</a></li>
    <li><a href="https://kmr.bioinformatics.nz">kmr.bioinformatics.nz</a></li>
    <li><a href="https://keybase.io/kmr">keybase.io/kmr</a></li>
    <li>Skype: kim_rutherford</li>
    </ul>
      </footer>
</body>
</html>
